src/components/LoginForm.jsx

```diff
@@ -0,0 +1,153 @@
+import React, { useState } from "react";
+import { Tab, Tabs, TextField, Stack, Button, Container } from "@mui/material";
+import { Box } from "@mui/system";
+
+const passwordMatchingText = {
+  error: "Passwords do not match",
+  valid: "",
+};
+
+function LoginForm() {
+  const [currentTabIndex, setCurrentTabIndex] = useState(0);
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [passwordVerification, setPasswordVerification] = useState("");
+  const [passwordMatchingMsg, setPasswordMatchingMsg] = useState(
+    passwordMatchingText.valid
+  );
+
+  const handleTabChange = (event, tabIndex) => {
+    setCurrentTabIndex(tabIndex);
+    setEmail("");
+    setPassword("");
+    setPasswordVerification("");
+    setPasswordMatchingMsg(passwordMatchingText.valid);
+  };
+
+  const handleEmailChange = (event) => {
+    setEmail(event.target.value);
+  };
+
+  const handlePasswordChange = (event) => {
+    setPassword(event.target.value);
+    setPasswordMatchingMsg(passwordMatchingText.valid);
+  };
+
+  const handlePasswordVerificationChange = (event) => {
+    setPasswordVerification(event.target.value);
+    setPasswordMatchingMsg(passwordMatchingText.valid);
+  };
+
+  /*
+    TODO:
+    - Handle api status code (200 vs != 200)
+    */
+  const onLogin = (event) => {
+    event.preventDefault();
+    alert("Welcome " + email);
+  };
+
+  /*
+    TODO:
+    - Handle api status code (200 vs != 200)
+    */
+  const onCreateAccount = (event) => {
+    event.preventDefault();
+    if (password !== passwordVerification) {
+      setPasswordMatchingMsg(passwordMatchingText.error);
+    }
+  };
+
+  return (
+    <Container
```

Comment:
In reference to my [earlier comment about responsiveness](https://github.com/secure-password-manager/PasswordManagerWeb/pull/13#discussion_r1375509280), the `minWidth` setting does help, but I took a look at the `mui` docs and what I think what I'm talking about is using [`<Grid>` components](https://mui.com/material-ui/react-grid/) to set up breakpoints.

This might be something that comes later, but as the screen gets skinnier I'd like to see the Login/Signup button get wider relative to the box, and eventually for the padding between the box and the edge of the window to disappear.

Reply:
Hey Derth! Yes sorry I should have clarified that point when I added minWidth. Breakpoints is the way to go. I didn't set up breakpoints here we'd be setting up when we have a page built out - this would be a single component

Reply:
That makes sense, you won't really know where to put the breakpoints until you see the component in context.